core/challenges.go

```diff
@@ -5,31 +5,73 @@
 
 package core
 
+import (
+	"encoding/json"
+	"github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/letsencrypt/go-jose"
+)
+
+func newChallenge(challengeType string, accountKey *jose.JsonWebKey) (Challenge, error) {
+	ak := AuthorizedKey{
+		Token: NewToken(),
+		Key:   accountKey,
+	}
+
+	jsonAK, err := json.Marshal(ak)
+	if err != nil {
+		return Challenge{}, err
+	}
+
+	return Challenge{
+		Type:          challengeType,
+		Status:        StatusPending,
+		AccountKey:    accountKey,
+		AuthorizedKey: jsonAK,
+	}, nil
+}
+
+//----- BEGIN TO DELETE -----
```

Comment:
Remove this and its pair and instead drop some links to issue tickets to remove them in these functions godoc. The tickets can be more clear as to when and why we'd delete them.

These comments as is will make people who don't know about this change nervy in the future when they come across it. "Should we delete this now? How did it get in??" that kind of thing.